ads/adtelligent.md

```diff
@@ -0,0 +1,59 @@
+<!---
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Adtelligent
+
+## Examples
+
+### For DSP/SSP Clients
+
+```
+<amp-embed width="300" height="250"
```

Comment:
do you want `amp-ad` or `amp-embed`?
usually all ad networks should go with `amp-ad`. content recommendation can use `amp-embed`. To use amp-embed, you will need to add an entry in AMP_EMBED_ALLOWED (integration.js)

Reply:
i'd like to stick with amp-ad

Reply:
please update this example then